index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <!-----Title Starts----->
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <!----------title changed--------->
+    <title>Homepage -HORISEON Social Solution Services</title>
+  </head>
+  <!----Title End---->
+
+  <!----Body start---->
+  <body>
+    <!-----Header---->
+    <div class="header">
```

Comment:
There is a semantic tag that you can use here. Take a look at [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more information about the header tag.

Reply:
Hi Chelsea, sorry for the late reply. I have made some of the changes you have requested. Right now I am happy with my grade so will not be asking for a re-grade. Thank you for the suggestions as they are really helpful.